urlbar-sayings.txt

```diff
@@ -231,3 +231,4 @@ http://canadianholidaysexplained.tumblr.com/
 〆(・∀・＠)
 (/) (°,,°) (/)
 moz://a
+"(Pull Requests|Patches|Commits) Welcome" translates quite cleanly to "Piss Off" in many languages
```

Comment:
I like "Go Away", I can do that.

If you can believe it, this is already three steps in the family-friendliness direction from the original :S